operations/finance.md

```diff
@@ -545,6 +642,104 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 * `Receipt` - the bill has already been fully paid.
 * `Invoice` - the bill is supposed to be paid in the future. Before closing it is balanced with an invoice payment.
 
+## Add bill
+
+Creates new empty bills assigned to account.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/bills/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "Bills": [
+        {
+            "AccountId": "a5786a7b-a388-43cc-a838-abd7007b5ff7",
+        },
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Bills` | array of [Bill parameters](#bill-parameters) | required | Information about bills to be created. |
+
+#### Bill parameters
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the bill is issued to. |
+
+
+### Response
+
+```javascript
+{
+    "Bills": [
+        {
+            "Id": "177966b7-f3d9-42b7-ba49-abd80057329b",
+            "CustomerId": "a5786a7b-a388-43cc-a838-abd7007b5ff7",
```

Comment:
Shoudnt we also have here account it ? @siroky  ?  as its more future proof ? 
Because they are providing account id but when get back there is customer id and company id

Reply:
AccountId definitely.